pkg/synchronization/controller.go

```diff
@@ -452,6 +476,33 @@ func (c *Controller) createOrUpdateServiceMetadata(logger *zap.Logger, ns *core_
 	return false, nil
 }
 
+// createOpsGenieSecretSpec creates an OpsGenieIntegrations secret object suitable for Secret creation
+func createOpsGenieSecretSpec(ogInt *opsgenieIntegrationManager.IntegrationsResponse, ns *core_v1.Namespace) (core_v1.Secret, error) {
+	jData, err := json.Marshal(ogInt)
+	if err != nil {
+		return core_v1.Secret{}, err
+	}
+
+	secretData := map[string][]byte{
```

Comment:
Why not inline it into the struct literal?

Reply:
Just testing you ;)